### PR TITLE
Update dev packages and  remove the additional jest install step

### PR DESCRIPTION
**Pull request by siemiatj**
Related to #9841 . We've had all the dependencies frozen in `packages.json` but during the build process in CI there was a separate installation of `jest` test runner, which automatically installed the latest version. Since we're already using it, there's not really a need to do that (and also having a fixed version defined in the `packages` we know that it'll work :)). Let me know if my thinking is not correct.

Also updated a bunch of packages.

### CODE REVIEW

**Comment by petrican** [comment]
> Related to #9841 . We've had all the dependencies frozen in `packages.json` but during the build process in CI there was a separate installation of `jest` test runner, which automatically installed the latest version. Since we're already using it, there's not really a need to do that (and also having a fixed version defined in the `packages` we know that it'll work :)). Let me know if my thinking is not correct.
> 
> Also updated a bunch of packages.

Your thinking is correct. Need to do some testing with this and will merge after. Thanks

**Review by petrican — APPROVED** [review]
lgtm. Thanks!

**Comment by petrican** [comment]
Screenshot 2020-08-26 at 09 46 03
There's seem to be an issue that I spot after merging this. There's this warning that we need to fix also @siemiatj

File: frontend/buildfile.groovy [hunk 1]
```diff
@@ -55,7 +55,6 @@ Map build(final MvnConf mvnConf, final Map scmVars, final boolean forceBuild=fal
 		sh 'yarn install'
 		sh 'yarn lint --quiet'
 							
-		sh "yarn add jest jest-junit --dev"
```
Comment (metas-ts) [inline frontend/buildfile.groovy:58]: @siemiatj fine with me! :+1:

**Pull request merged**